core/shared/src/main/scala/scodec/bits/ByteVector.scala

```diff
@@ -319,6 +319,32 @@ sealed abstract class ByteVector extends BitwiseOperations[ByteVector, Long] wit
     reverse.foldLeft(z)((tl,h) => f(h,tl))
 
   /**
+    * Applies a binary operator to a start value and all segments(views) of this ByteVector expressed as read-only ByteBuffer, going left to right.
+    * @param z    Starting value
+    * @param f    operator to apply
+    */
```

Comment:
Add `@group collection`  here and on foldRightBB -- this project uses ScalaDoc groups so every member needs a group annotation.